### DISCUSSION TITLE
from coqui-ai VITS TTS  to torchscript model convert

### DISCUSSION THREAD

**Discussion by bobur-amirov** [rhasspy/piper#469] [Q&A]
Can I convert Couqui VITS model to torchscript model with export_torchscript.py ?
Has anyone tried this before?

**Comment by synesthesiam** [comment]
I haven't tried this before, but the input and output parameters would have to match exactly for it to work.

**Reply by bobur-amirov** [reply]
Thanks for the reply!
I haven't tried a piper based train model yet.
So is the torchscript model for mobile faster than the .onnx model?
Which one do you recommend for offline use on mobile?
